src/org/labkey/test/util/APIAssayHelper.java

```diff
@@ -295,4 +297,49 @@ public Protocol copyAssayDesign(Integer assayId, String containerPath, String as
         ProtocolResponse saveProtocolResponse = saveProtocolCommand.execute(connection, containerPath);
         return saveProtocolResponse.getProtocol();
     }
+
+    public void createAssayWithPlateSupport(String name)
+    {
+        ReactAssayDesignerPage assayDesigner = createAssayDesign("General", name);
+
+        assayDesigner.setPlateMetadata(true);
+        assayDesigner.clickFinish();
+    }
+
+    public String getPlateTemplateLsid(String folderPath) throws Exception
+    {
+        SelectRowsCommand selectRowsCmd = new SelectRowsCommand("assay.General", "PlateTemplate");
+        selectRowsCmd.setColumns(List.of("Lsid"));
+
+        SelectRowsResponse resp = selectRowsCmd.execute(_test.createDefaultConnection(false), folderPath);
+
+        return String.valueOf(resp.getRows().get(0).get("Lsid"));
+    }
+
+    public void createPlateTemplate(String templateName, String templateType, String assayType)
+    {
```

Comment:
I don't think this method makes sense to move into this helper because it only creates a template with the same shape and has the weird behavior for GPAT template types. What would be better is to be able to parameterize the method so you could pass in size, well group and well location information. We could also consider whether it makes sense to push these into the PlateDesignerPage and have it create and return page objects like PlateWellGroup that in turn allows you to select wells for a particular well group.